demo/search_ranges.R

```diff
@@ -7,6 +7,7 @@ library(RXapian)
 db<-c("path/to/db")
 
 vrp1<-list(type="proc.num", value=c(0), check.str="mm", flags=c("RP_SUFFIX"))
+#If a Xapian version earlier than 1.3.6 is used replace vrp1 with: vrp1<-list(type="proc.num", value=c(0), check.str="mm", prefix=FALSE)
```

Comment:
Done.Could you kindly check if we can merge this PR now?

On Wed, Aug 10, 2016 at 2:30 AM, James Aylett <notifications@github.com>
wrote:

> In demo/search_ranges.R
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74143554>:
>
> > @@ -7,6 +7,7 @@ library(RXapian)
> >  db<-c("path/to/db")
> >
> >  vrp1<-list(type="proc.num", value=c(0), check.str="mm", flags=c("RP_SUFFIX"))
> > +#If a Xapian version earlier than 1.3.6 is used replace vrp1 with: vrp1<-list(type="proc.num", value=c(0), check.str="mm", prefix=FALSE)
>
> Having the API for R users be different depending on the version of Xapian
> used is awkward. You should be able to make the 1.4 version available for
> 1.2 users in this case. On 9 Aug 2016 9:25 p.m., amandaJayanetti <
> notifications@github.com> wrote:In demo/search_ranges.R:
> @@ -7,6 +7,7 @@ library(RXapian) db<-c("path/to/db")
> vrp1<-list(type="proc.num", value=c(0), check.str="mm",
> flags=c("RP_SUFFIX")) +#If a Xapian version earlier than 1.3.6 is used
> replace vrp1 with: vrp1<-list(type="proc.num", value=c(0), check.str="mm",
> prefix=FALSE)
> I didn't get you? … On Tue, Aug 9, 2016 at 3:22 PM, James Aylett
> ***@***.***> wrote: In demo/search_ranges.R <#25
> <https://github.com/amandaJayanetti/RXapian/pull/25> (comment)>: @@ -7,6
> +7,7 @@ library(RXapian) db<-c("path/to/db") vrp1<-list(type="proc.num",
> value=c(0), check.str="mm", flags=c("RP_SUFFIX")) +#If a Xapian version
> earlier than 1.3.6 is used replace vrp1 with: vrp1<-list(type="proc.num",
> value=c(0), check.str="mm", prefix=FALSE) For pre-1.3.6, you could
> explicitly check flags=c("RP_SUFFIX") yourself, to provide the same API to
> R users? — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub <https://github.com/
> amandaJayanetti/RXapian/pull/25/files/8b4adf893b924b6f2db7b7766841af
> 1266c7734f..4c44e39595bb06b87e1d5a33c427c21eef6ab74d#r74029463>, or mute
> the thread <https://github.com/notifications/unsubscribe-auth/AH_
> jNkoJ3wJ9rJir6MaQsOv8L0nbANEtks5qeE3NgaJpZM4Jf4Xo> . —You are receiving
> this because you commented.Reply to this email directly, view it on GitHub,
> or mute the thread.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/8b4adf893b924b6f2db7b7766841af1266c7734f..4c44e39595bb06b87e1d5a33c427c21eef6ab74d#r74143554>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNmqWQtPaYmdi_vy-VtgnijyerPuNks5qeOqFgaJpZM4Jf4Xo>
> .
>